src/server/admin/tags/controllers/add-tag.js

```diff
@@ -0,0 +1,29 @@
+import { addTagToService } from '~/src/server/admin/tags/helpers/fetchers.js'
+import { boomify } from '@hapi/boom'
+import Joi from 'joi'
+import { validServiceTags } from '~/src/server/admin/tags/helpers/service-tags.js'
+
+export const addTagController = {
+  options: {
+    validate: {
+      payload: Joi.object({
+        tag: validServiceTags.required(),
+        service: Joi.string().min(1).required()
+      }).unknown(true)
+    }
+  },
+  handler: async (request, h) => {
+    const { service, tag } = request.payload
+
+    request.logger.info(`Added ${tag} tag to ${service}`)
+
+    try {
+      await addTagToService(tag, service)
```

Comment:
this is a bit of an anti pattern. Your catching the error to just rethow it. I'd just not catch it